metricbeat/module/system/network_summary/_meta/fields.yml

```diff
@@ -0,0 +1,26 @@
+- name: network_summary
+  type: group
+  release: beta
+  description: >
+    Metrics relating to global network activity
+  fields:
+    - name: ip.*
+      type: object
+      description: >
+        IP protocol counters
```

Comment:
nit: {IP|TCP|UDP} contains "Protocol" in name, so there is a redundancy here: Internet Protocol protocol counters

(same for others)